nunit.sln

```diff
@@ -1,8 +1,6 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio 2012
```

Comment:
There are things I don't like about 2013, so I use 2012 for NUnit and only use 2013 for the adapter, which requires it.

We should aim to have our own releases constantly improve on their predecessors, unlike some other folks... ;-)